x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_multi_step_extension.tsx

```diff
@@ -6,13 +6,110 @@
  */
 
 import React, { memo } from 'react';
-import type { PackagePolicyCreateMultiStepExtensionComponentProps } from '@kbn/fleet-plugin/public';
+import { EuiText, EuiIcon, EuiFlexGroup, EuiFlexItem, EuiLink, EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import styled from 'styled-components';
 
 /**
- * TBD: A component to be displayed in multi step onboarding process.
+ * A component to be displayed in multi step onboarding process.
  */
-export const EndpointPolicyCreateMultiStepExtension =
-  memo<PackagePolicyCreateMultiStepExtensionComponentProps>(({ newPolicy }) => {
-    return <></>;
-  });
+export const EndpointPolicyCreateMultiStepExtension = memo(() => {
+  return (
+    <Container>
+      <LargeCustomSpacer />
+      <Title />
+      <MediumCustomSpacer />
+
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false} style={{ alignItems: 'center' }}>
+          <LargeSecurityLogo />
+        </EuiFlexItem>
+
+        <EuiFlexItem>
+          <div>
+            <Features />
+          </div>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+
+      <MediumCustomSpacer />
+      <Details />
+      <LargeCustomSpacer />
+    </Container>
+  );
+});
 EndpointPolicyCreateMultiStepExtension.displayName = 'EndpointPolicyCreateMultiStepExtension';
+
+const Container = styled('div')`
+  padding: 0 23px;
```

Comment:
Yeah, why is this exact padding needed instead of using the Eui token values for padding? Also looks like either the `Container` should have padding that is equal to the `LargeCustomSpacer` instead of the custom spacer. Also, the `EuiFlexGroup` on line 23  should have a margin that is equal to `MediumCustomSpacer` instead.

Reply:
c53eed7